dimagi/commcare-hq
# MVP map in case tile list

## Product Description
Add a map to case list if one of the properties is formatted as Address. Pin have popups that list all the fields in the table.

<img width="939" alt="image" src="https://github.com/dimagi/commcare-hq/assets/1946138/1a2f55ff-d6b4-44a9-b0c0-664c7ad1b78b">

<img width="1178" alt="image" src="https://github.com/dimagi/commcare-hq/assets/1946138/e442764b-122c-4ea1-92e9-c304bba5e6c5">


## Technical Summary
[Add the most basic possible map to the case list](https://dimagi-dev.atlassian.net/browse/USH-3199)

No form player changes. Just frontend

## Feature Flag
No feature flags are needed but there needs to be one property in the case list that is formatted as "Address".

## Safety Assurance

### Safety story
Tested locally

### Automated test coverage
No automated tests for the ui changes since it uses a third party api. Updated the tests that cover creation of the suite.xml

### QA Plan

Configure case list with and without "Address" property. Verify case list still works as expected for both normal case lists and case list tiles.

### Rollback instructions


### Labels & Review
